examples/container-service-extension-3.1.x/3.8-cse-3.1.x-install.tf

```diff
@@ -1,3 +1,8 @@
+# ------------------------------------------------------------------------------------------------------------
```

Comment:
Starting to review, but can't seem to find the changelog entry?

Reply:
Added! I just added the installation part, the following PR will update it with the cluster management changelog entry.